### ISSUE TITLE
Feed into JW Player

### ISSUE THREAD

**Issue by 123mijnwebsite** [jwplayer/appletv#47]
Hello guys!

I'm still looking for a way to insert a json feed so jw player will extract all those video's en content and show it in the app.

Like:

`[{"id":"805ca389-3729-11e6-9a28-5254005f012e","version":"14","slug":"lezentv-amandakluveld-a-20120611","name":"Amanda Kluveld","synopsisshort":"Amanda Kluveld: Pijn - interview","synopsislong":"Historica Amanda Kluveld schreef een indrukwekkende en zeer toegankelijke studie over de culturele, de maatschappelijke en de occulte betekenis van pijn door de eeuwen heen en deed bovendien een aantal persoonlijke ontdekkingen. \u201cPijn heeft iets moois zolang die beheersbaar is.\u201d","url":"http:\/\/lezentv.nl\/articles\/136","genre":"Overige, Serie","duration":"00:32:45","duration_seconds":"1965","starttime":"2016-06-24 04:02:32","endtime":"2016-06-24 04:35:17","date":"2016-06-23"}`

So basically it got all the information it needs to show the right video, but I just don't understand how to get this to work in the app? 

I know I need to change the Json file but not really know how. I was hoping someone can help me and tell me what I exactly need to do to get this to work?

(Is it also possible to insert this feed in my JW Player account like a playlist and it extract all the videos from it? I can't find any information about this either)

Thanks for helping!

**Comment by josiekeller** [comment]
Hi there -- the Apple TV App Builder is built to work with JW Player-hosted video content, which is why your JSON file isn't loading video into the app. You can use the Platform API to upload your content to JW Platform, which will make it compatible with the app builder: https://developer.jwplayer.com/jw-platform/reference/v1/uploads.html.
